paper/paper.Rmd

```diff
@@ -35,6 +35,10 @@ affiliations:
     name: Preisenergie GmbH, Munich, Germany
   - index: 5
     name: Jumping Rivers
+  - index: 6
+    name: The University of Queensland
+  - index: 7
+    name: Unknown
```

Comment:
Temporary entry to make sure that PDF can be built: https://github.com/openjournals/joss-reviews/issues/7171#issuecomment-2327379155